src/test/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionaryIntegrationTest.java

```diff
@@ -46,4 +46,17 @@ public void testNonUniqueSequenceName() throws Exception {
         runCommandLine(argv);
         Assert.fail("Exception should have been thrown.");
     }
+
+    // Should throw an exception because no reference file was specified
+    @Test(expectedExceptions = {UserException.CommandLineException.class})
+    public void testCustomCommandLineValidation() throws Exception {
```

Comment:
Name this test case better -- something like `testRunWithoutReference` instead of `testCustomCommandLineValidation`